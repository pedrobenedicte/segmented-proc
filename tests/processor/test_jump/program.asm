add r0 <- r1, r2
add r3 <- r4, r5
add r6 <- r7, r1
add r1 <- r4, r5
bnz r7, 10
add r2 <- r4, r5
add r1 <- r2, r5
add r2 <- r2, r5
nop
nop
nop
