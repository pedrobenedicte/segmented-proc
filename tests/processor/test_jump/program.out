0a
40
e5
40
b9
41
37
60
65
40
a5
40
55
40
95
40
00
00
00
00
00
00
